clusterloader2/pkg/measurement/common/probes/manifests/metricsServer/metrics-server-prober-role.yaml

```diff
@@ -0,0 +1,14 @@
+---
+apiVersion: rbac.authorization.k8s.io/v1
```

Comment:
What's the point of splitting into multiple files? Keeping whole BRA-related stuff in one file as it was actually is pretty convinient to use...

Reply:
Because I found that some resources were not created successfully  in one file, so splitting into multiple files.
And I found that dnsLookup and network are also separated

Reply:
ok, I have I have rollbacked to use one file.